tests/e2e/e2e_test.go

```diff
@@ -88,6 +92,82 @@ func (s *IntegrationTestSuite) TestIBC() {
 	s.testIBCTokenTransfer()
 	s.testMultihopIBCTokenTransfer()
 	s.testFailedMultihopIBCTokenTransfer()
+
+	// stop hermes0 to prevent hermes0 relaying transactions
+	s.Require().NoError(s.dkrPool.Purge(s.hermesResource0))
+	HermesResource0Purged = true
+	s.testIBCBypassMsg()
+}
+
+func (s *IntegrationTestSuite) testIBCBypassMsg() {
+
+	// submit gov proposal to change bypass-msg param to
+	// ["/ibc.core.channel.v1.MsgRecvPacket",
+	//  "/ibc.core.channel.v1.MsgAcknowledgement",
+	//  "/ibc.core.client.v1.MsgUpdateClient"]
+	submitterAddr := s.chainA.validators[0].keyInfo.GetAddress()
+	submitter := submitterAddr.String()
+	proposalCounter++
+	s.govProposeNewBypassMsgs([]string{
+		sdk.MsgTypeURL(&ibcchanneltypes.MsgRecvPacket{}),
+		sdk.MsgTypeURL(&ibcchanneltypes.MsgAcknowledgement{}),
+		sdk.MsgTypeURL(&ibcclienttypes.MsgUpdateClient{})}, proposalCounter, submitter, standardFees.String())
+
+	// use hermes1 to test default ibc bypass-msg
+	//
+	// test 1: transaction only contains bypass-msgs, pass
+	s.T().Logf("testing transaction contains only ibc bypass messages")
+	ok := s.hermesTransfer(hermesCofigWithGasPrices, s.chainA.id, s.chainB.id, transferChannel, uatomDenom, 100, 1000, 1)
+	s.Require().True(ok)
+
+	scrRelayerBalanceBefore, dstRelayerBalanceBefore := s.queryRelayerWalletsBalances()
+
+	pass := s.hermesClearPacket(hermesCofigNoGasPrices, s.chainA.id, transferChannel)
+	s.Require().True(pass)
+	pendingPacketsExist := s.hermesPendingPackets(hermesCofigNoGasPrices, s.chainA.id, transferChannel)
+	s.Require().False(pendingPacketsExist)
+
+	// confirm relayer wallets do not pay fees
+	scrRelayerBalanceAfter, dstRelayerBalanceAfter := s.queryRelayerWalletsBalances()
+	s.Require().Equal(scrRelayerBalanceBefore.String(), scrRelayerBalanceAfter.String())
+	s.Require().Equal(dstRelayerBalanceBefore.String(), dstRelayerBalanceAfter.String())
+
+	// test 2: test transactions contains both bypass and non-bypass msgs (ibc msg mix with time out msg)
+	s.T().Logf("testing transaction contains both bypass and non-bypass messages")
+	ok = s.hermesTransfer(hermesCofigWithGasPrices, s.chainA.id, s.chainB.id, transferChannel, uatomDenom, 100, 1, 1)
+	s.Require().True(ok)
+	// make sure that the transaction is timeout
+	time.Sleep(3)
+	pendingPacketsExist = s.hermesPendingPackets(hermesCofigNoGasPrices, s.chainA.id, transferChannel)
+	s.Require().True(pendingPacketsExist)
+
+	pass = s.hermesClearPacket(hermesCofigNoGasPrices, s.chainA.id, transferChannel)
+	s.Require().False(pass)
+	// clear packets with paying fee, to not influence the next transaction
+	pass = s.hermesClearPacket(hermesCofigWithGasPrices, s.chainA.id, transferChannel)
+	s.Require().True(pass)
+
+	// test 3: test transactions contains both bypass and non-bypass msgs (ibc msg mix with time out msg)
```

Comment:
Does this comment match the code? The code seems to test the case where you cannot clear without gas because it exceeds the max bypass